routes.js

```diff
@@ -5,7 +5,7 @@ const path = require('path')
 
 function fastifySwagger (fastify, opts, next) {
   fastify.route({
-    url: `${opts.prefix}/json`,
+    url: '/json',
```

Comment:
@mcollina Seems like when using `fastify-plugin` it's ignoring the route prefix

Reply:
there is no need to use fastify-plugin in ./routes.js

Reply:
Got it, I was confused why it was used there to begin with.

I'll take it out.